test/repository_test.rb

```diff
@@ -68,6 +68,16 @@
             UserRepository.persist(persisted_user)
             persisted_user.created_at.must_equal persisted_user.created_at
           end
+
+          it 'update updated_at' do
+            update_user = UserRepository.find(user.id)
+            update_user.name = 'My'
+
+            UserRepository.persist(update_user)
```

Comment:
What do you think about reassign the result into the `updated_user` var? That is `updated_user = UserRepository.persist(update_user)`